queries/commonlisp/highlights.scm

```diff
@@ -0,0 +1,142 @@
+(sym_lit) @variable
+
+;; A highlighting for functions/macros in th cl namespace is available in theHamsta/nvim-treesitter-commonlisp
+;(list_lit . (sym_lit) @function.builtin (#cl-standard-symbol? @function.builtin))
```

Comment:
the plugin is basically only two tables with the standard symbols of Common Lisp